package.json

```
@@ -284,6 +285,28 @@
             "default": false,
             "description": "Disable the running of SourceKit-LSP.",
             "order": 6
+          },
+          "sourcekit-lsp.supported-languages": {
+            "type": "array",
+            "default": [
+              "swift",
+              "objective-c",
+              "objective-cpp",
+              "c",
+              "cpp"
+            ],
+            "description": "List of languages supported by SourceKit-LSP. This is used to determine when to start the SourceKit-LSP server.",
```

This isn't completely accurate. Maybe...
```suggestion
            "description": "List of languages supported by SourceKit-LSP. This is used to determine whether SourceKit-LSP should provide language features for a particular file type.",
```